.github/workflows/tooljet-docker-develop-build.yml

```diff
@@ -0,0 +1,49 @@
+name: Tooljet-develop Docker Image
+
+on:
+  pull_request:
+    branches:
+      - develop
+
+  workflow_dispatch:
+
+jobs:  
+  tooljet-develop-image:
```

Comment:
let's also build develop for tooljet server image

Reply:
done